sdk/go/auto/stack.go

```diff
@@ -1455,6 +1456,21 @@ func (fw *fileWatcher) Close() {
 
 	// Tell the watcher to end on next EoF, wait for the done event, then cleanup.
 
+	// The tail library we're using is racy when shutting down.
+	// If it gets the shutdown signal before reading the data, it
+	// will just shut down before finding the EoF.  This problem
+	// is exacerbated on Windows, where we use the poller, which
+	// polls for changes every 250ms.  Sleep a little bit longer
+	// than that to ensure the tail library had a chance to read
+	// the whole file.  On OSs that don't use the poller we still
+	// want to try to avoid the problem so we sleep for a short
+	// amount of time.
```

Comment:
Should we include a link to https://github.com/nxadm/tail/issues/67 in the comment?

Reply:
Good idea, done :heavy_check_mark: